src/app.ts

```diff
@@ -15,35 +15,6 @@ interface Flags {
   save: string;
 }
 
-interface CoinMarketResponse {
```

Comment:
We don't need to declare our own type here anymore. We added types to the API responses see: https://github.com/michael-siek/coingecko-api/pull/39